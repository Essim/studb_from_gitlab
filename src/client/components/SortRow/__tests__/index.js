import React from 'react';
import {SortRow} from '../';
import { shallowToJson } from 'enzyme-to-json';

describe('SortRow', () => {
    it('should render correctly', () => {
        const output = shallow(<SortRow/>);
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});
