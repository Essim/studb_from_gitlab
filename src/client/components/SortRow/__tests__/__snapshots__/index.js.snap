// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortRow should render correctly 1`] = `
<div
  className="row"
>
  <div
    className="col-xs-9"
  >
     
  </div>
  <div
    className="col-xs-3"
  >
    <div
      className="form-group form-inline"
    >
      Trier par Â 
      <select
        className="form-control"
      >
        <option>
          Nouveau
        </option>
        <option>
          Populaire
        </option>
        <option>
          Mes contributions
        </option>
      </select>
    </div>
  </div>
</div>
`;
